src/_pytest/stepwise.py

```diff
@@ -47,8 +47,6 @@ def pytest_sessionfinish(session: Session) -> None:
             # Do not update cache if this process is a xdist worker to prevent
             # race conditions (#10641).
             return
-        # Clear the list of failing tests if the plugin is not active.
```

Comment:
@davidszotten do you recall the rationale for this?

Ref https://github.com/pytest-dev/pytest/pull/4147

Reply:
pretty sure I just copied it from @nip3o 's original plugin. git blame leads to https://github.com/nip3o/pytest-stepwise/commit/e63037a4388a7dd124797d80353a062cc3a7b2ab (without much info unfortunately) 

though if removing this line, how does one reset the cache? while handy to have it remembered within a "working session", would it be unexpected to have it remembered across such sessions?

Reply:
> though if removing this line, how does one reset the cache? 

Good point, probably makes sense to add a `--stepwise-clear` flag for that.

Reply:
Added `--stepwise-clear`. :+1:

Reply:
Nice. Maybe -reset would be a clearer name than -clear? (Internally we are clearing a cache but conceptually it resets the stepwise stopping)

Reply:
Nice, great suggestion. Done.

Reply:
I think the original reason is what you mention @davidszotten . If you start a new test session where stepwise is active, it's confusing to have it jump back to a test that failed at some distant-or-not point in the past. I think clarifying the UI as suggested would be a reasonable solution.

Reply:
Thanks @nip3o!